src/UI/Tooltip.js

```diff
@@ -70,16 +79,18 @@ class Tooltip {
             'visibility': 'visible',
             'opacity': '1'
         }
+        // @ts-ignore how to narrow this.host to HTMLElement?
         addInlineStyles(this.host, forcedVisibilityStyles)
         this.input = input
         this.associatedForm = associatedForm
         this.interface = Interface
+        this.count = 0
     }
     append () {
         document.body.appendChild(this.host)
     }
     remove () {
-        window.removeEventListener('scroll', this.checkPosition, {passive: true, capture: true})
```

Comment:
I see why we removed this (["Note: The basic scroll event cannot be canceled"](https://developers.google.com/web/updates/2016/06/passive-event-listeners?hl=en)), but we should also remove it from where we add this listener. Do you mind double checking?

Reply:
Yep, I'll remove it. Actually I didn't remove it for that reason, I removed it because TS told me that option was not available when 'removing' an event :)